### PR TITLE
[rwt_image_view] use absolute path to load js and css

### PR DESCRIPTION
**Pull request by knorth55**
use absolute path to load js and css
js and css cannot be loaded in current relative path with `localhost:8000/rwt_image_view`.
(however they can be loaded with `localhost:8000/rwt_image_view/`)
I use absolute path to solve this problem.

cc. @708yamaguchi

### CODE REVIEW

File: rwt_image_view/www/index.html
Comment (k-okada) [inline rwt_image_view/www/index.html:40]: https://github.com/tork-a/visualization_rwt/pull/77/files changed absolute path to relative path. Which is correct?
https://github.com/tork-a/visualization_rwt/blob/959bf918d6c9d110fbf01a11298e0facafe1bf30/rwt_robot_monitor/www/plotter.html#L41
https://github.com/tork-a/visualization_rwt/pull/77/files#r259242919

**Comment by knorth55** [comment]
@k-okada 

## Current branch

I'm not familiar with javascript, so I'm not sure which one is correct.
However, with current code, we get this kind of load error when we access to `address:8000/rwt_image_view`
(not `address:8000/rwt_image_view/`)
this may be because javascript path is not `/rwt_image_view` but `/` when we access to `address:8000/rwt_image_view`.

rwt_image_view_current

### This PR

And when we change to load them with absolute path, we can solve the load error when we access to `address:8000/rwt_image_view`
(of course we can access to `address:8000/rwt_image_view/`)
rwt_image_view_new

### Note
However, I tried this in Kinetic, when we access to `address:8000/rwt_image_view`, it is automatically redirected to `address:8000/rwt_image_view/`, so this fix is related to some package version.
I don't know which package change the behavior, but we have this issue on Fetch15.

@iory 
can you tell me why you prefer relative path to absolute path?

File: rwt_image_view/www/index.html
Comment (knorth55) [reply rwt_image_view/www/index.html:40]: I'm writing the issue details here. I'm not familiar with javascript, so I don't know which one is better.
But in order to solve the issue, I use absolute path, but if we can solve the issue with relative path, we can use relative path probably.

https://github.com/tork-a/visualization_rwt/pull/91#issuecomment-649978444

**Comment by knorth55** [comment]
the current branch requires this PR (redirect from `rwt_image_view` -> `rwt_image_view/`) to avoid this error.
this PR is added in `roswww` `0.1.11`.
https://github.com/tork-a/roswww/pull/42

**Comment by knorth55** [comment]
@iory can you give us comment for this PR?

**Review by iory — APPROVED** [review]

**Comment by iory** [comment]
Which of `absolute path` and `relative path` is better depends on your preference. In this case, I like the former.
I think the maintainer should decide the rules.

**Pull request merged**
